src/azure-cli/azure/cli/command_modules/search/aaz/latest/search/service/_create.py

```diff
@@ -47,7 +47,7 @@ def _build_arguments_schema(cls, *args, **kwargs):
         )
         _args_schema.search_service_name = AAZStrArg(
             options=["-n", "--name", "--search-service-name"],
-            help="The name of the Azure Cognitive Search service to create or update. Search service names must only contain lowercase letters, digits or dashes, cannot use dash as the first two or last one characters, cannot contain consecutive dashes, and must be between 2 and 60 characters in length. Search service names must be globally unique since they are part of the service URI (https://<name>.search.windows.net). You cannot change the service name after the service is created.",
+            help="The name of the Azure Cognitive Search service to create or update. Search service names must only contain lowercase letters, digits or dashes, cannot use dash as the first two or last one characters, cannot contain consecutive dashes, and must be between 2 and 60 characters in length. Search service names must be globally unique since they are part of the service URI (`https://<name>.search.windows.net`). You cannot change the service name after the service is created.",
```

Comment:
Please fix it in aaz model as well

Reply:
search service didn't upload their original aaz models into cli repo.

Reply:
Please contact the search service to provide the aaz model and fix their models.